CHANGES.rst

```diff
@@ -1,21 +1,32 @@
 Changes
 =======
 
-Version 0.8.1 (UNRELEASED)
+Version 0.8.1 (2022-02-09)
 --------------------------
 
 - Users:
     - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+    - Adds Kubernetes job memory limits validation before publishing workflow submission.
+    - Adds support for HTML preview of workspace files in the user interface.
+    - Adds search by name in workflow file list page of the user interface.
+    - Changes user interface to display a more accurate Cluster Health status page, based on the availability instead of the usage.
+    - Changes ``info`` command to include the list of supported compute backends.
+    - Fixes workflow stuck in pending status due to early Yadage failures.
 - Administrators:
     - Adds configuration to set default job timeout value for the Kubernetes compute backend jobs (``kubernetes_jobs_timeout_limit``).
     - Adds configuration to set maximum job timeout that users can assign to their jobs for the Kubernetes compute backend (``kubernetes_jobs_max_user_timeout_limit``).
     - Adds new configuration option ``compute_backends`` to specify the supported list of compute backends for validation purposes.
     - Adds new configuration option ``reana_server.uwsgi.log_all`` to toggle the logging of all the HTTP requests.
     - Adds new configuration options ``reana_server.uwsgi.log_4xx`` and ``reana_server.uwsgi.log_5xx`` to only log HTTP error requests, i.e. HTTP requests with status code 4XX and 5XX. To make this configuration effective ``reana_server.uwsgi.log_all`` must be ``false``.
     - Adds new configuration options ``node_label_infrastructuremq`` and ``node_label_infrastructuredb`` to have the possibility to run the Message Broker and the Database pods in specific nodes.
+    - Adds workflow name validation to the ``create_workflow`` endpoint, restricting special characters like dots.
     - Changes uWSGI configuration to log all HTTP requests in REANA-Server by default.
-    - Changes ``.Values.quota.disk_update`` to ``.Values.quota.periodic_update_policy`` to also update the CPU quota. Keeps ``.Values.quota.disk_update`` for backward compatibility.
+    - Changes ``quota.disk_update`` to ``quota.periodic_update_policy`` to also update the CPU quota. Keeps ``quota.disk_update`` for backward compatibility.
     - Changes the name of configuration option ``quota.termination_update_policy`` to ``quota.workflow_termination_update_policy``. Keeps ``quota.termination_update_policy`` for backward compatibility.
+    - Changes ``/api/info`` endpoint to return a list of supported compute backends.
+    - Changes ``/api/status`` endpoint to calculate the cluster health status based on the availability instead of the usage.
+    - Changes the way of determining Snakemake job statuses, polling the Job Controller API instead of checking local files.
+    - Changes dependencies to solve compatibility issues with ``Yadage 0.20.2`` version.
```

Comment:
Hmm, we have 0.20.1 pinned in `r-w-e-yadage`, and 0.20.2 in `r-server`... So dunno how we would like to advertise this.
Changes of workflow versions are mostly in the "Users" section anyway. This seems kind of internal and can be left out here? Or else we should explain what we have in mind: client, cluster, or which component...

Reply:
Removed.